src/framework/preferences/js/StarterGrades.js

```diff
@@ -97,23 +97,15 @@ var fluid_2_0 = fluid_2_0 || {};
 
     fluid.defaults("fluid.uiEnhancer.starterEnactors", {
         gradeNames: ["fluid.uiEnhancer", "fluid.uiEnhancer.cssClassEnhancerBase", "fluid.uiEnhancer.browserTextEnhancerBase", "autoInit"],
-        connectionsGrade: "fluid.prefs.uiEnhancerConnections",
-        distributeOptions: {
-            source: "{that}.options.connectionsGrade",
-            removeSource: true,
-            target: "{that > fluid.prefs.enactor}.options.gradeNames"
-        },
+        model: "{fluid.prefs.rootModel}.rootModel",
```

Comment:
I've got a little lost here and forgotten what the "rootModel" component actually does - it appears that it acts here as a source of initial values (defaults?) for this grade. We should probably rename it to something a little more descriptive now that we have other models at the "root". Does "defaultRootModel" or "rootDefaultModel" seem reasonable?